pkg/cli/updater/updater.go

```diff
@@ -272,7 +272,7 @@ func (u *updater) check(ctx context.Context) (bool, error) {
 
 		// if we're not past the last update threshold
 		// then we don't check for updates.
-		if !time.Now().After(repoCache.LastChecked.Add(checkAmount)) {
+		if time.Now().Before(repoCache.LastChecked.Add(checkAmount)) {
```

Comment:
Can you revert this line? It's that horrible linter bug we fixed; just ignore the warning.

Reply:
Was this only fixed on RC? I am still seeing this error come up. I've added a no ling just in case.

Reply:
Possibly. I thought it had made it into the last release, but maybe not.